cs/src/Management/TunnelManagementClient.cs

```diff
@@ -533,45 +708,52 @@ public void Dispose()
             return BuildUri(
                 tunnel.ClusterId,
                 tunnelPath + (!string.IsNullOrEmpty(path) ? path : string.Empty),
-                options,
-                query);
+                query,
+                options);
         }
 
-        private async Task<AuthenticationHeaderValue?> GetAccessTokenAsync(
+        /// <summary>
+        /// 
```

Comment:
Empty doc comments.

Reply:
Oh, this one should be `private`. I had considered making it `protected` but there's no need. I'll change back to `private` and remove the empty doc-comments.